src/main/java/org/skriptlang/skript/lang/script/Script.java

```diff
@@ -39,11 +42,12 @@
  * Every script also has its own internal information, such as
  *  custom data, suppressed warnings, and associated event handlers.
  */
-public final class Script {
+public final class Script implements Experimented {
 
 	private final Config config;
 
 	private final List<Structure> structures;
+	private final Set<Experiment> experiments;
```

Comment:
I'd prefer to see experiments on scripts integrated using ScriptData instead rather than directly with this class.